app/page.tsx

```diff
@@ -2,29 +2,201 @@
 import { useAuth } from "./AuthContext";
 import TopBar from "@/components/TopBar";
 import BottomNavigationBar from "@/components/BottomNavigationBar";
+import { getSessionOccurrences, getSessions } from "./SessionList";
+import { useState, useEffect } from "react";
+import { LiveClasses } from "./types";
+import Link from "next/link";
+import PrimaryButton from "@/components/Button";
+import Loading from "./loading";
 
 export default function Home() {
   const { loggedIn, userId } = useAuth();
+  const [liveClasses, setLiveClasses] = useState<LiveClasses[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [quizzes, setQuizzes] = useState<LiveClasses[]>([]);
+  const baseUrl = process.env.NEXT_PUBLIC_AF_QUIZ_URL;
+  const apiKey = process.env.NEXT_PUBLIC_AF_QUIZ_API_KEY;
+  const userID = process.env.NEXT_PUBLIC_AF_QUIZ_USER_ID;
+
+  const fetchSessionOccurrencesAndDetails = async () => {
+    try {
+      const sessionOccurrenceData = await getSessionOccurrences();
```

Comment:
Session occurence list will be very huge na? It's a huge table and we're fetching all of it. Is it fine?

Reply:
Yeah, we can make optimizations in an another PR